roles/setup_assisted_installer/defaults/main.yml

```diff
@@ -9,18 +9,18 @@ assisted_service_image: "quay.io/ocpmetal/assisted-service:{{ assisted_service_t
 assisted_service_gui_image: "quay.io/ocpmetal/ocp-metal-ui:{{ assisted_service_gui_tag }}"
 assisted_postgres_image: quay.io/ocpmetal/postgresql-12-centos7
 
-assisted_service_openshift_versions:
+assisted_service_openshift_versions_defaults:
   "4.6":
     display_name: 4.6.16
-    release_image: quay.io/openshift-release-dev/ocp-release:4.6.16-x86_64
+    release_image: "quay.io/openshift-release-dev/ocp-release@{{ image_hashes['release_4.6'] | default('4.6.16-x86_64') }}"
```

Comment:
Is an image name that uses a default value still valid after `:` was replaced with `@`?

Reply:
Good spot :)